vendor/assets/javascripts/brainstem/brainstem-model.coffee

```diff
@@ -95,6 +105,16 @@ class window.Brainstem.Model extends Backbone.Model
     else
       super(field)
 
+  set: (key, val, options) ->
+    super
+
+    _.extend(@unsavedAttributes || {}, @changed)
```

Comment:
Okay, but in the case where we use the {} here, don't those not get extended into anything that we're keeping a reference to?

---
Andrew Cantino
Chief Architect, Mavenlink

> On Nov 20, 2015, at 3:14 PM, Jamie Rolfs <notifications@github.com> wrote:
> 
> In vendor/assets/javascripts/brainstem/brainstem-model.coffee:
> 
> > @@ -95,6 +105,16 @@ class window.Brainstem.Model extends Backbone.Model
> >      else
> >        super(field)
> >  
> > +  set: (key, val, options) ->
> > +    super
> > +
> > +    _.extend(@unsavedAttributes || {}, @changed)
> No, it kind of breaks @unsavedAttributes in the case of instantiating the model with attributes (kind of edge case because usually you don't pass id in which case all attributes would be persisted).
> 
> —
> Reply to this email directly or view it on GitHub.
>